### PR TITLE
add caveat of Ubuntu Advantage or Landscape seats for on-prem

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* added caveat to the /support/plans-and-pricing page for Landscape On-Premises server
* updated the copy doc

## QA

1. go to /support/plans-and-pricing
2. compare the Landscape text with the [copy doc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#)

## Screenshots

image

### CODE REVIEW

File: templates/support/plans-and-pricing.html [hunk 1]
```diff
@@ -244,7 +244,7 @@ <h3 class="row-pricing__title">Ubuntu Advantage Desktop</h3>
     <div class="strip-inner-wrapper">
         <div class="ten-col">
             <h2>Landscape on-premises server</h2>
-            <p>If you&rsquo;d prefer a server on premises running behind your firewall, Landscape can be installed on private hardware. This is functionally identical to the SaaS, but you control the hardware.</p>
+            <p>If you&rsquo;d prefer a server on premises running behind your firewall, Landscape can be installed on private hardware. This is functionally identical to the SaaS, but you control the hardware. Requires all the desktop, servers or nodes it manages to have either a Ubuntu Advantage or Landscape seat.</p>
```
Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: This requires? As it's a continuation of the paragraph.

Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: desktop*s*, servers and nodes

Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: "an Ubuntu Advantage or Landscape seat."?

Comment (pmahnke) [inline templates/support/plans-and-pricing.html]: all good catches... I think the first one is fine... I updated the other two per your suggestions.

File: templates/support/plans-and-pricing.html [hunk 2]
```diff
@@ -244,7 +244,7 @@ <h3 class="row-pricing__title">Ubuntu Advantage Desktop</h3>
     <div class="strip-inner-wrapper">
         <div class="ten-col">
             <h2>Landscape on-premises server</h2>
-            <p>If you&rsquo;d prefer a server on premises running behind your firewall, Landscape can be installed on private hardware. This is functionally identical to the SaaS, but you control the hardware.</p>
+            <p>If you&rsquo;d prefer a server on premises running behind your firewall, Landscape can be installed on private hardware. This is functionally identical to the SaaS, but you control the hardware. Requires all the desktop, servers and nodes it manages to have either an Ubuntu Advantage or Landscape seat.</p>
```
Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: s/desktop/desktops

Comment (pmahnke) [inline templates/support/plans-and-pricing.html]: you are on FIRE!

**Pull request merged**